app/src/main/java/com/bignerdranch/android/androidacademy/data/MoviesLoad.kt

```diff
@@ -0,0 +1,56 @@
+package com.bignerdranch.android.androidacademy.data
+
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import retrofit2.Retrofit
+import retrofit2.http.GET
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import retrofit2.create
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+class MoviesRep {
+    suspend fun loadMovies(): Page {
+        return RetrofitModule.moviesApi.getMovies()
+    }
+
+    suspend fun loadDetails(id: Long): MovieDetails {
+        return RetrofitModule.moviesApi.getMovieDetails(id)
+    }
+
+    suspend fun loadActors(id: Long): Credit {
+        return RetrofitModule.moviesApi.getActors(id)
+    }
+}
+
+private interface MoviesApi {
```

Comment:
we agreed that to separate Movies api and retrofit module to another file

Reply:
movies api moved to another file